app/vmauth/auth_config.go

```diff
@@ -289,11 +291,14 @@ func initAuthConfig() {
 	// See https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1240
 	sighupCh := procutil.NewSighupChan()
 
-	m, err := readAuthConfig(*authConfigPath)
+	ac, m, err := readAuthConfig(*authConfigPath)
 	if err != nil {
 		logger.Fatalf("cannot load auth config from `-auth.config=%s`: %s", *authConfigPath, err)
 	}
-	authConfig.Store(m)
+	if err = ipfilters.Init(ac.IPFilters); err != nil {
```

Comment:
It is better to do not add `ipfilters` package into open-source version of `vmauth`, since it is just a stub, which may confuse users who read open-source code of `vmauth`. It is better to try to localize the `ipfilters` functionality as much as possible into a separate package / files, and minimize the diff inside open-source files. This will improve maintainability of enterprise-only functionality and will minimize potential merge conflicts between open source and enterprise source code in the future.